### PR TITLE
Triton 1987: Prevent exception when query.type is not set

### PR DESCRIPTION
**Pull request by kusor**

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -524,10 +522,13 @@ function createServer(options) {
                 var size_histogram = collector.getCollector(
                     METRIC_SIZE_HISTOGRAM);
 
-                var labels = { type: query.type() };
-                query_counter.increment(labels);
-                latency_histogram.observe(lat / 1000, labels);
-                size_histogram.observe(query.bytesSent, labels);
+                var queryType = query.type();
+                if (queryType) {
+                        var labels = { type: query.type() };
```
Comment (mgerdts) [inline lib/server.js]: Why have queryType if it is only going to be used once?  Did you mean to reuse it on line 527?

File: lib/server.js [hunk 2]
```diff
@@ -524,10 +522,13 @@ function createServer(options) {
                 var size_histogram = collector.getCollector(
                     METRIC_SIZE_HISTOGRAM);
```
Comment (mgerdts) [inline lib/server.js]: Line 516 - 523 could be in the `if` block.

File: lib/server.js (hunk 1 above)
Comment (kusor) [reply lib/server.js]: Indeed, modifying in a bit.

File: lib/server.js (hunk 2 above)
Comment (kusor) [reply lib/server.js]: Indeed. Thanks for the review. Applied the suggested changes and pushed again.

**Pull request merged**